### PR TITLE
Some adjustments after UX review

### PR DESCRIPTION
**Pull request by reno1979**
- FileList inside FileUpload shown below dropzone
- FileUpload font 18px
- Image has max-width of 50px to prevent overflow

image

image

### CODE REVIEW

File: src/form/FileUpload/FileUploadStyles.ts [hunk 1]
```diff
@@ -16,9 +16,14 @@ export const FileUploadStyle = styled.div`
...
 
+export const FileUploadPlaceholderWrapper = styled.div`
+	display: flex;
+	gap: ${themeSpacing(2)};
+`;
+
 export const FileUploadPlaceholderTextCss = css`
 	font-weight: 400;
-	font-size: 24px;
+	font-size: 18px;
```
Comment (dlnr) [inline src/form/FileUpload/FileUploadStyles.ts:26]: Font size but not the line-height? This already conflicts the DS. I would prefer it if this was inherited from the body or set by variables.

File: src/form/FileUpload/FileUploadStyles.ts [hunk 2]
```diff
@@ -16,9 +16,14 @@ export const FileUploadStyle = styled.div`
 	cursor: pointer;
 `;
 
+export const FileUploadPlaceholderWrapper = styled.div`
+	display: flex;
```
Comment (dlnr) [inline src/form/FileUpload/FileUploadStyles.ts:20]: No direction or wrapping?

**Review by dlnr — APPROVED** [review]
Maybe it's up to @HeeChanVanDerHaar to check if that font size thing doesn't look weird in DMS.

File: src/form/FileUpload/FileUploadStyles.ts (hunk 2 above)
Comment (reno1979) [reply src/form/FileUpload/FileUploadStyles.ts:20]: default flex direction, it is meant for a line of text of which a part is hidden at a certain resolution

File: src/form/FileUpload/FileUploadStyles.ts (hunk 1 above)
Comment (reno1979) [reply src/form/FileUpload/FileUploadStyles.ts:26]: Ok, noted. We will make some better decisions about font styling and how to apply them soon

**Pull request merged**